models/nulog/NuLogParser.py

```diff
@@ -267,7 +267,11 @@ def get_test_dataloaders(self, data_tokenized, transform_to_tensor):
     def load_data(self, windows_folder_path):
         try:
             df_log = self.log_to_dataframe(windows_folder_path)
-            return [df_log.iloc[i].Content for i in range(df_log.shape[0])]
+            return [
+                df_log.iloc[i].Content
+                for i in range(df_log.shape[0])
+                if df_log.iloc[i].Content is not None
+            ]
```

Comment:
can you make this return more pandas efficient?

Reply:
addressed.